### PR TITLE
Release 1.2.43

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-0000](https://jira.cms.gov/browse/MAT-0000)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Comment by sb-cecilialiu** [comment]
Regular width modal: 
image

vs wider modal for HR:
image

File: react/components/Modal/Modal.jsx [hunk 1]
```diff
@@ -30,6 +42,9 @@ const Modal = ({
             overlayClassName="qpp-c-modal__overlay"
             bodyOpenClassName="qpp-c-modal__body-open"
             isOpen={isOpen}
+            style={{
+                content: getWidth(),
```
Comment (nmorasb) [inline react/components/Modal/Modal.jsx:46]: just to confirm, this has been tested/verified with other places aside from HR where the modal is being used that this style is desired?

Comment (sb-cecilialiu) [reply react/components/Modal/Modal.jsx:46]: @nmorasb Modal is only used in ExportModal, and as long as it's not passing in a width, it should be used as a regular modal (as shown in the first modal in the above screenshots). So the regular modal still works as before, for HR we'll need to pass in a width.

Comment (nmorasb) [reply react/components/Modal/Modal.jsx:46]: got it, sounds good!
